fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -3009,6 +3014,37 @@ public CommandProcessingResult makeRefund(final Long loanId, final LoanTransacti
                 .build();
     }
 
+    @Override
```

Comment:
LoanWritePlatformService getting bloated. Would you consider to move this into its own service?